README.md

```diff
@@ -208,6 +217,28 @@ Uses of `oBrandGet` within `oBrandConfigureFor` [retrieves a variable for a vari
 }
 ```
 
+### Override The Current Brand
+
+It may be desirable to output a customised variant of a component based on brand configuration. To do this create a component specific mixin which maps arguments to `oBrandOverride`. This prevents brand configuration from becoming a public interface, which makes it possible to change brand variables at a later point without a breaking change.
```

Comment:
Could we reword this to something along lines of, 'If a variant requires customisation beyond brand configuration, it can be applied through a  component specific mixin.... ' ?